rbac/management/group/definer.py

```diff
@@ -168,6 +178,8 @@ def add_roles(group, roles_or_role_ids, tenant, user=None):
         system_policy.roles.add(role)
 
         dual_write_handler = RelationApiDualWriteGroupHandler(group, ReplicationEventType.ASSIGN_ROLE, [])
+        if relations is not None:
+            dual_write_handler.extend_relations_to_remove(relations)
```

Comment:
Seems the relations to remove is not complete?
We have to 
1. remove the the default_bindings_from_mapping generated for the system platform_default group;
2. add the 
default_bindings_from_mapping generated for the custom platform_default group;
3. remove all the users to the system platform_default
4. add all the users to the custom platform_default

Reply:
@astrozzc Step is there but I am not sure about step 2-4.

My understanding was that all those bindings are using some default custom uuid from mapping.

Reply:
Thanks for Libor's walking through offline, I have some misunderstanding of some steps. So we don't need the 3 and 4